### PR TITLE
installed axios

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by jeremyjorand — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/package.json [hunk 1]
```diff
@@ -10,10 +10,12 @@
     "fix": "prettier --write . && eslint --fix --ext .jsx ."
   },
   "dependencies": {
+    "axios": "^0.26.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "styled-components": "^5.3.5",
-    "styled-reset": "^4.3.4"
+    "styled-reset": "^4.3.4",
+    "template-fullstack": "file:.."
```
Comment (LoicBrassart) [inline frontend/package.json:18]: Nope ! :warning:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
There's a sus package in your package.json ! :scream:

File: frontend/package.json (hunk 1 above)
Comment (ThomasDCW) [reply frontend/package.json:18]: Ok, what is the solution it's seems like the styled reset is installed twice ?

**Pull request closed without merging**